### PR TITLE
Job to check Matterbridge processes

### PR DESCRIPTION
**Pull request by eneiluj**
refs #4097

Start the ones that should be running, kill the ones that are not related with a room in config.

### CODE REVIEW

File: lib/BackgroundJob/CheckMatterbridges.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Talk\BackgroundJob;
+
+use OC\BackgroundJob\TimedJob;
```
Comment (nickvergessen) [inline lib/BackgroundJob/CheckMatterbridges.php:26]:
```suggestion
use OCP\BackgroundJob\TimedJob;
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]
Use public api as commented

**Comment by eneiluj** [comment]
I took the liberty to fix this for other background jobs (where I got the error from :grin:).

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**